internal/controller/restore/reconciler.go

```diff
@@ -131,6 +150,8 @@ func (r *SingleRestoreReconciler) checkRestoreStatus() error {
 	}
 
 	r.Log.Info(fmt.Sprintf("Restore status: %+v", restoreStatus))
+	r.Recorder.Eventf(r.aeroRestore, corev1.EventTypeNormal, "RestoreStatus",
+		"Restore status: %+v", restoreStatus)
```

Comment:
Seems too much info for event. Better not to add in events.
Instead add Restore Compete event in the main reconcile func at the very end https://github.com/aerospike/aerospike-kubernetes-operator/pull/324/files#diff-47218f09e6b90f3189bfd8f3e13c64f77d5ac7bf48c9039ff21ee499ff944f33R73